.github/workflows/ci.yml

```diff
@@ -11,7 +11,7 @@ jobs:
       - name: Setup Sage
         uses: einride/sage/actions/setup@master
         with:
-          go-version: "1.22"
+          go-version: "1.23"
```

Comment:
Might want to add the flag to check for updates otherwise newer patch versions will cause govuln to complain